server/features/content_rewrite.feature

```diff
@@ -89,13 +153,63 @@ Feature: Rewrite content
         Then we get array of _items by _id
         """
         {
-            "#firstassignment#": {"assigned_to": {"state": "in_progress"}}
+            "#firstassignment#": {"assigned_to": {"state": "completed"}}
+        }
+        """
+        When we rewrite "#archive._id#"
+        """
+        {"desk_id": "#desks._id#"}
+        """
+        Then we get OK response
+        When we get "/planning/#planning._id#"
+        Then we get OK response
+        When we get "/archive/#REWRITE_ID#"
+        Then we get existing resource
+        """
+        {
+            "_id": "#REWRITE_ID#",
+            "assignment_id": "#firstassignment#"
         }
         """
+        When we get "/published"
+        Then we get existing resource
+        """
+        {"_items" : [{"_id": "123", "rewritten_by": "#REWRITE_ID#", "assignment_id": "#firstassignment#"}]}
+        """
+        When we publish "#REWRITE_ID#" with "publish" type and "published" state
+        Then we get OK response
+        When we get "/published"
+        Then we get list with 2 items
+        """
+        {
+        "_items": [
+            { "_id": "123", "rewritten_by": "#REWRITE_ID#", "assignment_id": "#firstassignment#" },
+            {"_id": "#REWRITE_ID#", "rewrite_of": "123", "assignment_id": "#firstassignment#"}
+        ]}
+        """
+        When we enqueue published
+        When we get "/publish_queue"
```

Comment:
As this is critical change, can we have end-to-end behave test which includes publishing of `planning item` using json planning formatter. (
see feature https://github.com/superdesk/superdesk-planning/blob/master/server/features/assignments_planning_publish.feature#L610
see step https://github.com/superdesk/superdesk-planning/blob/master/server/features/assignments_planning_publish.feature#L953
)
In the step you can assert the deliveries as well.